### DISCUSSION TITLE
Laravel模型关联及Amis列表数据字段对应问题

### DISCUSSION THREAD

**Discussion by YouLiLiao** [slowlyo/owl-admin#112] [开发建议]
Laravel模型关联，根据规范来说都会使用驼峰命名法，然而返回数据给前端时会自动转为蛇形命名法。这时候owl-admin列表自动处理关联数据时就会有问题了，正如下图所示，我有个上级用户`fromUser`的关联，列表的`TableColumn`的`name`需对应`fromUser`才会自动处理关联数据，然而返回给前端时，`fromUser`变成了`from_user`，这时候就导致列表无法找到对应的字段。
![image](https://github.com/slowlyo/owl-admin/assets/11286354/9f91efda-af65-4c86-8bd2-1b21b7e9a4f6)
![image](https://github.com/slowlyo/owl-admin/assets/11286354/b62f33af-112f-490a-a7b7-18230b2487f1)
![image](https://github.com/slowlyo/owl-admin/assets/11286354/566a4231-fe1d-460e-bba6-71af77502b3f)
![image](https://github.com/slowlyo/owl-admin/assets/11286354/7935145e-b614-437f-8f45-a9447c724de8)

**Comment by slowlyo** [comment]
确实没注意   后续优化一下

**Reply by YouLiLiao** [reply]
目前只能通过关联钩子来处理了，还有哥，你文档什么时候能完善一下，逼得只能去翻源码了o(╥﹏╥)o
![image](https://github.com/slowlyo/owl-admin/assets/11286354/f4c8f659-2b36-461f-a7bf-5a6301edefba)
![image](https://github.com/slowlyo/owl-admin/assets/11286354/a5ab5254-22e5-4f95-9584-960cccf4540f)

**Reply by slowlyo** [reply]
欢迎pr https://github.com/owl-admin/docs
